tests/peak_finder_test.py

```diff
@@ -0,0 +1,63 @@
+#  Copyright (c) 2021 Mira Geoscience Ltd.
+#
+#  This file is part of geoapps.
+#
+#  geoapps is distributed under the terms and conditions of the MIT License
+#  (see LICENSE file at the root of this source code package).
+
+
+import tempfile
+from pathlib import Path
+
+import numpy as np
+from geoh5py.objects import Curve
+from geoh5py.workspace import Workspace
+
+from geoapps.processing.peak_finder import PeakFinder
+
+
+def test_peak_finder_app():
```

Comment:
pytest comes with a builtin tmp path fixture.  Just add tmp_path to your tests arguments: def test_peak_finder_app(tmp_path):

Reply:
Oh very nice! Done